appinventor/components/src/com/google/appinventor/components/runtime/ChartDataBase.java

```diff
@@ -0,0 +1,1006 @@
+// -*- mode: java; c-basic-offset: 2; -*-
```

Comment:
TODO: Possibly combining DataFileColumns and WebColumns into one could work out better than having two separate entries.